src/View/SSViewer.php

```diff
@@ -822,17 +895,29 @@ public function setTemplateFile($type, $file)
      * @param string $contentGeneratedSoFar The content of the template generated so far; it should contain
      * the DOCTYPE declaration.
      * @return string
+     * @deprecated 5.4.0 Use getBaseTag() instead
      */
     public static function get_base_tag($contentGeneratedSoFar)
```

Comment:
Renaming this, and passing in a boolean so it's easier to use by other engines.

But for now our template parser will keep calling this (hence why it's suppressed). I've opted to not add `Deprecation::withSuppressedNotice()` anywhere inside the template parser, so anything that calls has a suppressed notice directly in the method being called.